cluster/addons/kube-ui/kube-ui-rc.yaml

```diff
@@ -1,27 +1,27 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: kube-ui-v2
+  name: kube-ui-v3
```

Comment:
I think I was copying another example when I put this together, but now that I know the system better, is there anyreason to include the version in the lables & name like this? I think we could probably just rely on the image version.